### ISSUE TITLE
Missing buttons+

### ISSUE THREAD

**Issue by hcloud00** [xternal7/ultrawidify#11]
The tool buttons in the youtube player are now gone(after the update) and the video keeps witching between 16:9 and 21:9 by itself (however it often chooses the wrong aspect ratio)

**Comment by TangoFox4** [comment]
Yes, they are missing. They were bugging sometimes, stretching and stuff. Now they are gone. I liked them.

**Comment by F900EX** [comment]
I have the same issue, buttons are gone in YT and the does not work for me.

**Comment by xternal7** [comment]
Buttons are gone for good and I don't plan on adding them back for the time being.

Reason for the decision is that they're too much work to maintain or add into the player UI. If I want to add a different site later down the line, the part that adds buttons to the player UI often needs to be coded for each site individually.

I'll look into the aspect ratio switching issue later today when I return home. Are there any specific videos where this happens or does it happen across the board? More importantly, does it happen on youtube as well?

Keyboard shortcuts still work, whereas switching via popup might be broken for random reasons. 

Aspect ratio switching works in full screen only at the moment.

**Comment by TangoFox4** [comment]
https://www.youtube.com/watch?v=XAWGbEzzkyw
Example. Works for me.

**Comment by xternal7** [comment]
Automatic aspect ratio switching works for this video as (well for) me. (Although I did get an example of broken auto-ar switching in #12 — but not to the degree described in this ticket)

@hcloud00 please be a bit more specific and include links to the videos where the issue happens. If the video is long, please also include timestamps. 

@F900EX Your description is rather vague. By "not working", do you mean you have the same issue with aspect ratio switching between 16:9 and 21:9 for no reason? If so, please include some links because I'd love to check that out.

**Comment by hcloud00** [comment]
@xternal7 the issue occurs when the video it self becomes 21:9 briefly. The plugin will recognize that and switch the to ultra-wide but will lag to switch back to 16:9 even though it eventually switches back. The following video shows what i mean goto time stamp; 4:16

p.s. thanks for putting in the work and making this plugin :)

**Comment by xternal7** [comment]
Can you post the link again please? For some reason it didn't post.

**Comment by hcloud00** [comment]
https://www.youtube.com/watch?v=gEbldRT9uaM&t=261s

sorry forgot to add it

**Comment by F900EX** [comment]
After uninstall and reinstalling it, it works fine on full screen.  Thanks.
